### PR TITLE
Assignment1_Amith_Udupa_G

### PR DESCRIPTION
**Pull request by Amith2604**

### CODE REVIEW

File: subit.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                <p><input id='1' type="checkbox">I have no taste.I am pretty dense so your site doesn't do much for</p>
+            </div>
+        </div>
+        <div class="'choose">
+            <div id="choose"><b>Choose the thing you like the most in my website</b></div>
+            <div class="dido">
+                <p><input id='2' type="radio" name="a">All the gorgeous picture</p>
+                <p><input id='2' type="radio" name="a">All the gorgeous picture of ur cat</p>
+                <p><input id='2' type="radio" name="a">The inspiring recap of our amazing suburban childhood</p>
+                <p><input id='2' type="radio" nmae="a">The detailled list of all of our ellvis memorabillia's</p>
```
Comment (hrushikeshj) [inline subit.html:40]: `id` has to be unique

**Review by hrushikeshj — COMMENTED** [review]
Hey, good job but make sure all week 1 HTML files are in the [week_1](https://github.com/IRIS-NITK/IRIS-RoR-Bootcamp-2022/tree/main/week_1) folder.

File: subit.html (hunk 1 above)
Comment (Amith2604) [reply subit.html:40]: Ok,Should i resubmit it for you to approve it or will you approve it.if should re submit how?

Comment (hrushikeshj) [reply subit.html:40]: You push push your changes if you want. Refer to [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork), for how to push changes.

**Pull request closed without merging**